modules/apps/portal-language/portal-language-override-web/src/main/java/com/liferay/portal/language/override/web/internal/display/context/ViewDisplayContextFactory.java

```diff
@@ -193,6 +194,7 @@ private List<LanguageItemDisplay> _getAllLanguageItemDisplays(
 					}
 				}
 
+				Collections.sort(overrideLanguageIds);
```

Comment:
If you search for `overrideLanguageIds`, you'll find we use this list in another spot as well. Since both of those lists are passed into the `_getLanguageIdsString` method, it's best to sort once there and cover all cases.